io/src/pcd_grabber.cpp

```diff
@@ -387,7 +387,6 @@ pcl::PCDGrabberBase::PCDGrabberBase (const std::vector<std::string>& pcd_files,
 ///////////////////////////////////////////////////////////////////////////////////////////
 pcl::PCDGrabberBase::~PCDGrabberBase () throw ()
 {
-  stop ();
   delete impl_;
```

Comment:
We might need to be more careful here because the copy constructor and assignment operator of `PCDGrabberBase` both copy this raw pointer. Would wrapping it into a shared pointer be a good idea?